packages/filefeeds-react/src/OneSchemaFileFeeds.tsx

```diff
@@ -143,6 +148,8 @@ export default function OneSchemaFileFeeds({
   // Launch props
   customizationKey,
   customizationOverrides,
+  // Init Session props
```

Comment:
Can we move this to be the first item under the "primary props" section, considering that it overrides almost any other prop there?